docs/reference/known_limitations.md

```diff
@@ -15,25 +15,35 @@ We are developing a new feature that will allow you to modify these settings dir
 
 ## Backups
 
-The limitations related to backups in Percona Everest are as follows:
+Let's delve into the limitations of on-demand backups in Percona Everest. 
+
+### PostgreSQL limitations for on-demand backups
 
 - When attempting to delete a PostgreSQL database that contains backups created with Everest versions older than 1.0.0, the database may become stuck in the **Deleting** state. 
 
     **Workaround**: To prevent this, manually delete any backups created with versions prior to 1.0.0 by using the **Delete** action on the **Backups** page before deleting the database.
 
 - You cannot change the bucket name for a specific backup storage. Doing so will make any backups taken for that bucket unusable.
 
+- You can create on-demand backups using only the first three backup storages if there are more than three backup storages.
+
+    For example, if you create backup storages everest-1, everest-2, everest-3, and everest-4 sequentially, you can only use everest-1, everest-2, and everest-3 for on-demand backups.
 
 
 ## Scheduled backups
 
-Due to an issue that has been identified in PostgreSQL editing/deletion of schedules, the following functionality is temporarily unavailable for PostgreSQL:
+Let's explore the constraints of scheduled backups in Percona Everest.
 
-- Modifying the storage location in existing schedules
-- Removing existing schedules
-- Utilizing the same storage location for different schedules
+### PostgreSQL Limitations for schedules
 
-The bug may result in a situation where editing or deleting the schedules could corrupt previously taken backups, making it impossible to restore from them.
+Due to PostgreSQL limitations, the following functionality is unavailable for PostgreSQL:
+
+- Modifying the storage location in existing schedules
+- Using the same backup storage for different schedules
+- Creating more than three schedules for PostgreSQL
+- Creating a schedule that uses the fourth backup storage when three backup storages are already in use.
```

Comment:
the last two are not exactly true, the truth is: 

- Using more than 3 different storages in total. The total number of used storages includes the storages used in the existing backups and schedules.